app/templates/index.hbs

```diff
@@ -3,16 +3,16 @@
 We hope you find exactly what you're looking for in a place to stay.
 <br/><br/>
 
-{{filter-listing
-  filteredList=filteredList
-  autoComplete=(action 'autoComplete')
-  search=(action 'search')}}
-
-<ul>
-  {{#each model as |rentalUnit|}}
-    {{rental-listing rental=rentalUnit}}
-  {{/each}}
-</ul>
+{{#filter-listing
+   filter=(action 'filterByCity')
+   findAll=(action 'findAllRentals')
```

Comment:
@toddjordan Here you pass in the action `findAllRentals` from the index controller, but `findAll` is never invoked within the `filter-listing` component.